app/models/translation.rb

```diff
@@ -116,22 +116,22 @@ def name_desc_onesky(phrases)
   def create_translated_attributes(phrases)
     translation_attribute_ids = []
     resource.translated_attributes.each do |translated_attribute|
-      translation = phrases[translated_attribute.onesky_phrase_id]
+      translation = phrases[translated_attribute.crowdin_phrase_id]
 
       if translation.present?
         translation_attribute = translation_attributes.where(key: translated_attribute.key).first_or_initialize
         translation_attribute.update(value: translation)
         translation_attribute_ids << translation_attribute.id
       elsif translated_attribute.required
-        raise Error::TextNotFoundError, "Translated phrase not found: ID: #{translated_attribute.onesky_phrase_id}"
+        raise Error::TextNotFoundError, "Translated phrase not found: ID: #{translated_attribute.crowdin_phrase_id}"
       end
     end
 
     translation_attributes.where.not(id: translation_attribute_ids).delete_all
   end
 
   def download_translated_phrases(filename)
```

Comment:
The `filename` param is no longer used below; but without that, I don't understand how `translated_page` and `translated_tip` will work properly. It looks like they are supposed to work against a specific `page.filename` or `"tip_#{tip.name}.json"`.